Documentation/rendering.md

```diff
@@ -35,7 +35,11 @@ However, you can also just keep the special URL **builtin://Engine** to load the
 
 ## Loading a model from Azure Storage
 
-From the **Model** dropdown menu, select **Load from storage...** and select a [converted](conversion.md) `.arrAsset` file from your connected storage account.
+From the **Model** dropdown menu, select **Load from storage...** and select a [converted](conversion.md) `.arrAsset` file from your connected storage account. In the background, ARRT will generate a [SAS token](https://docs.microsoft.com/azure/storage/common/storage-sas-overview) from which to load the model.
+
+## Loading a model from Azure Storage connected to an ARR Account
+
+If you have an [ARR Account connected to a Storage Account](https://learn.microsoft.com/en-us/azure/remote-rendering/how-tos/create-an-account#link-storage-accounts), a [SAS token](https://docs.microsoft.com/azure/storage/common/storage-sas-overview) isn't needed to load the model. From the **Model** dropdown menu, select **Load from storage linked to ARR account...** and select a [converted](conversion.md) `.arrAsset` file from your connected storage account.
 
```

Comment:
The first path still has "/en-us/" in it. Please remove

Reply:
thanks, fixed!